copyright_and_licenses.csv

```diff
@@ -145,6 +145,8 @@ is_lmi_ISN2004_ISN2016.tif,"2017-2019, National Land Survey of Iceland",CC-BY-4.
 is_lmi_ISN93_ISN2016.tif,"2017-2019, National Land Survey of Iceland",CC-BY-4.0
 is_lmi_ISN_vel_beta.tif,"2017-2019, National Land Survey of Iceland",CC-BY-4.0
 is_lmi_README.txt,Disclaimed,Public domain
+jp_gsi_gsigeo2011_mainland.tif,Geospatial Information Authority of Japan,CC-BY-4.0,測量法に基づく国土地理院長承認(使用)R 2JHs 501
```

Comment:
This is a CSV file, so we can't really add a column. The extra mention is probably good enough in the readme

Reply:
Fixed